lib/amazon_mws.rb

```diff
@@ -12,7 +12,7 @@
 
 class AmazonMwsOrderResponse
   def initialize(response)
-    @response = response.fetch("GetOrderReferenceDetailsResponse", {})
+    @response = Hash.from_xml(response.body).fetch("GetOrderReferenceDetailsResponse", {})
```

Comment:
Maybe I'm just confused on this one. Only thing I was suggesting was to wrap the response in the newly created `Response` class, but maybe they are just different _responses_